libraries/dawg-show/src/main/java/com/comcast/video/dawg/show/PowerController.java

```diff
@@ -89,7 +94,14 @@ public void power(@RequestParam(value="deviceIds[]") String[] deviceIds, @Reques
     }
 
     protected PowerClient getPowerClient(MetaStb stb, Cookie authCookie) {
-        PowerClient pc = new PowerClient(stb);
+        MetaStb meta = new MetaStb(new HashMap<String, Object>(stb.getData())); // make a copy so we can set some defaults
```

Comment:
considered it, pretty low amount of repeated code. But maybe it makes enough sense.